examples/src/multiple_kms_cmk_regions.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""
+Example showing basic encryption and decryption of a value already in memory
+using multiple KMS CMKs in multiple regions.
+"""
+import aws_encryption_sdk
+
+
+def encrypt(kms_key_provider, source_plaintext):
+    """Encrypts source_plaintext with the key(s) in kms_key_provider"""
+    return aws_encryption_sdk.encrypt(source=source_plaintext, key_provider=kms_key_provider)
+
+
+def decrypt(kms_key_provider, ciphertext):
+    """Decrypts ciphertext with the key(s) in kms_key_provider"""
+    return aws_encryption_sdk.decrypt(source=ciphertext, key_provider=kms_key_provider)
+
+
+def multiple_kms_cmk_regions(key_arn1, key_arn2, source_plaintext, botocore_session=None):
+    """Encrypts and then decrypts a string under multiple KMS customer master keys (CMKs) in multiple regions.
+
+    :param str key_arn1: Amazon Resource Name (ARN) of the KMS CMK
+    :param str key_arn2: Amazon Resource Name (ARN) of another KMS CMK
+    :param bytes source_plaintext: Data to encrypt
+    :param botocore_session: existing botocore session instance
+    :type botocore_session: botocore.session.Session
+    """
+    # Check that these keys are in different regions
+    assert not key_arn1[12:21] == key_arn2[12:21]
```

Comment:
This is brittle. Partition names are not always the same length and region names are not always the same length.

Arns are, however, always colon-delimited and ordered the same way, so you can split and pull out the region.

ex:
```python
region = key_arn1.split(":")[3]
```